projects/RPi/options

```diff
@@ -148,7 +148,7 @@
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-psx /}"
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-saturn /}"
     LIBRETRO_CORES="${LIBRETRO_CORES// beetle-vb /}"
-    LIBRETRO_CORES="${LIBRETRO_CORES// bsnes_hd /}"
+    LIBRETRO_CORES="${LIBRETRO_CORES// bsnes2014 /}"
```

Comment:
shouldn't it be ` bsnes ` which should be removed from the list?

Reply:
Indeed, let me fix it

Reply:
Actually both are removed, see line 153